docs/src/api/class-fetchrequest.md

```diff
@@ -6,6 +6,12 @@ environment or the service to your e2e test. When used on [Page] or a [BrowserCo
 the cookies from the corresponding [BrowserContext]. This means that if you log in using this API, your e2e test
 will be logged in and vice versa.
 
+## async method: FetchRequest.dispose
+
+Disposes this request, it will release all responses received for this request. Call this method when you are done
```

Comment:
`All responses received through [fetch], [get], [post] and other methods are stored. in the memory, so that you can later call [FetchResponse.body]. This method discards all stored responses, and makes [FetchResponse.body] throw with "Response disposed" error.`

Reply:
Updated text.